cpu/xive.py

```diff
@@ -19,6 +19,8 @@
 
 from avocado import Test
 from avocado.utils import genio
+from avocado.utils import distro
+from avocado.utils import process
```

Comment:
I think you are not using the process, so remove this.

Reply:
Sure, done!

> I think you are not using the process, so remove this.